app/assets/javascripts/mapping_rules.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function () {
```

Comment:
So we are isolating this script and including it in the new view.... We might need to create new issues for:
1. Merge with Cadu's work
2. Update this ugly script with a proper component